projects/plugins/jetpack/_inc/lib/core-api/wpcom-endpoints/class-wpcom-rest-api-v2-endpoint-joltw.php

```diff
@@ -0,0 +1,102 @@
+<?php
+/**
+ * joltw - To be named. jetpack#19212
+ * Jetpack interface for calling atomic-auth-proxy on wpcom.
+ *
+ * @package automattic/jetpack
+ * @since ???
+ */
+
+use Automattic\Jetpack\Connection\Client;
+
+/**
+ * joltw - Call wpcom atomic-auth-proxy from jetpack.
+ *
+ * @since ???
+ */
+class WPCOM_REST_API_V2_Endpoint_Joltw extends WP_REST_Controller {
+	/**
+	 * Constructor.
+	 */
+	public function __construct() {
+		$this->namespace = 'wpcom/v2';
+		$this->rest_base = 'joltw';
+		add_action( 'rest_api_init', array( $this, 'register_routes' ) );
+	}
+
+	/**
+	 * Register the route.
+	 */
+	public function register_routes() {
+		register_rest_route($this->namespace, '/' . $this->rest_base . '/file(?P<path>/.+)?', [
+			[
+				'show_in_index' => false,
+				'methods' => WP_REST_Server::READABLE,
+				'callback' => [ $this, 'fetch_remote_media_file' ],
+				'permission_callback' => [ $this, 'permission_check' ],
+				'args' => [],
```

Comment:
`path`, sizes etc. would go here.

Reply:
my initial goal was to be able to retrieve data (any data) with a hardcoded site, path, etc and then work on handling these properly